### ISSUE TITLE
u32x8 compare methods implemented as functions and not as traits

### ISSUE THREAD

**Issue by RazrFalcon** [Lokathor/wide#85]

**Comment by Lokathor** [comment]
say what now?

**Comment by RazrFalcon** [comment]
https://github.com/Lokathor/wide/blob/cb91288ded8f2c2101cd23f2a64bede67240f4e3/src/u32x8_.rs#L254

https://github.com/Lokathor/wide/blob/cb91288ded8f2c2101cd23f2a64bede67240f4e3/src/u32x4_.rs#L177

**Comment by Lokathor** [comment]
Oh. but it looks like those methods also exist in the latest of the main branch? And in the published 0.6.4 version.

Where are you *not* seeing them available?

**Comment by RazrFalcon** [comment]
Yes, they exist. But in f32x8 they are implemented [as traits](https://github.com/Lokathor/wide/blob/cb91288ded8f2c2101cd23f2a64bede67240f4e3/src/f32x8_.rs#L309), not as methods.

**Comment by Lokathor** [comment]
Oh, that, yeah. If you want to have the trait option just make a PR for trait impls that call the method. We can get that published real quick.

**Comment by RazrFalcon** [comment]
No, I don't actually use them. Just noticed this issue while was reading the code. It can be fixed at any time.
